actions/vm_bestfit.yaml

```diff
@@ -0,0 +1,38 @@
+---
+name: vm_bestfit
+runner_type: python-script
+description: >
+    This action is used to determine the best host and datastore to provision a new VM to on a given cluster. The result host will be the one with the least amount of VMs and the result datastore will be the one with the most free space.
+enabled: true
+entry_point: vm_bestfit.py
+parameters:
+    cluster_name:
+        type: string
+        description: Name of the cluster in vSphere
+        required: true
+    datastore_filter_regex:
+        type: array
+        items:
+          type: string
+        description: >
+          'Regular expression to filter the list of available datastores
```

Comment:
this example could be improved "string1" and "string2" aren't that interested
also should note that when the regex matches a datastore name, then it will NOT be considered or have VMs added to it